sdk/metricsadvisor/ai-metrics-advisor/samples/javascript/incidentsAndAlerts.js

```diff
@@ -53,14 +53,15 @@ async function listIncidentsForDetectionConfig(client, detectionConfigId) {
     console.log(`      detection config id: ${incident.detectionConfigurationId}`);
   }
 
+  const pageSettings = { maxPageSize: 20 };
   console.log(`  by pages`);
   const iterator = client
     .listIncidentsForDetectionConfiguration(
       detectionConfigId,
       new Date("09/06/2020"),
       new Date("09/11/2020")
     )
-    .byPage({ maxPageSize: 20 });
+    .byPage(pageSettings);
```

Comment:
this is fine. The work item is mostly about create/update methods

Reply:
ooh. Is it ok to update this or should i undo it?

Reply:
undone